### PR TITLE
imported camera aspect set

### PR DESCRIPTION
**Pull request by Suprhimp**
Hi!

I found that the imported camera from the uploaded model has wrong camera aspect.

So, I think it will be better to add some setting that adjust camera aspect to imported camera.

I changed this below.

update for Imported camera view

add code in setCamera for active camera aspect set

change code in resize function to adjust window resize

Thanks!

### CODE REVIEW

File: src/viewer.js [hunk 1]
```diff
@@ -175,9 +175,9 @@ export class Viewer {
 
     const {clientHeight, clientWidth} = this.el.parentElement;
 
-    this.defaultCamera.aspect = clientWidth / clientHeight;
-    this.defaultCamera.updateProjectionMatrix();
-    this.vignette.style({aspect: this.defaultCamera.aspect});
```
Comment (donmccurdy) [inline src/viewer.js:180]: Unfortunately I think it's unclear what the right result would be here. The camera embedded in the file is likely to have its own aspect ratio, in which case I'm not sure we should override it. 

_Ideally_ I'd like to keep the camera's aspect ratio, but "letterbox" the scene, with [renderer.setViewport](https://threejs.org/docs/?q=webglr#api/en/renderers/WebGLRenderer.setViewport) or something similar. Then we could render with the camera's intended aspect ratio, without distorting what the user sees.

Comment (Suprhimp) [reply src/viewer.js:180]: Oh I got what you mean.

I thought that as I modeling some project and put in the gltf viewer, I felt un comfortable to use it.
Cause it looks distorted.

I didn't thought that there are modeler set camera aspect ratio.

But just my opinion is the user will thought strange if the camera aspect is different with webpage.
:)

Comment (donmccurdy) [reply src/viewer.js:180]: Yeah, I think both are true. We shouldn't stretch the camera to fit the webpage. But we also shouldn't ignore the camera's aspect ratio. Letterboxing is the way to go here, but I haven't implemented it so far.

Comment (Suprhimp) [reply src/viewer.js:180]: Than maby I have to close this pull request until Letterboxing (The check box for the user whether change camera aspect ratio as I understand ) implemented.

Or I can add Check box for that.

Comment (donmccurdy) [reply src/viewer.js:180]: Yeah sorry! I think just letterboxing is the way I'd prefer to go.

**Pull request closed without merging**